gamestonk_terminal/README.md

```diff
@@ -323,7 +323,7 @@ Command|Description|Source
 `independence`  |tests independent and identically distributed (BDS) | [Wikipedia](https://en.wikipedia.org/wiki/Pearson%27s_chi-squared_test#Testing_for_statistical_independence)
 
 &nbsp;
-## FRED [»](fred/README.md)
+## FRED [»](econ/README.md)
 Command|Description
 ------ | --------
 `gdp`           | GDP
```

Comment:
Here you should add the view command for vix, and even a 2 additional rows to distingusih between FRED and VIX